examples/src/python/example/python_distribution/hello/fasthello/BUILD

```diff
@@ -19,5 +19,6 @@ python_binary(
   source='main.py',
   dependencies=[
     ':fasthello',
-  ]
```

Comment:
is it possible to specify `platform=` on python_dist as well?

Reply:
It is currently not supported. I cannot recall why we did not but I remember there being a reason for not supporting it, at least initially.